Statistical_Inference/Variance/lesson

```diff
@@ -263,7 +263,7 @@
   Hint: Remember the many many examples we went through. The sqrt(n) figured prominently in them.
 
 - Class: text
-  Output: Congrats! You've concluded this vary long lesson on variance. We hope you liked it vary much.
```

Comment:
Yes, I got that after covering other lessons of Statistical Inference. I'm sorry for such a bother.